starter-code/src/contactsList.js

```diff
@@ -994,4 +994,13 @@
     "pictureUrl": "https://image.tmdb.org/t/p/w500/aqZ9RjL5j44HMlBMvTaawhHiGOH.jpg",
     "popularity": 6.914606
   }
-]
\ No newline at end of file
+]
+
+
+export function getContacts() {
```

Comment:
As mentioned before, very nice to export functions like this and call them in any component where you need it. 👍